dojo/tools/awssecurityhub/compliance.py

```diff
@@ -7,13 +7,21 @@ def get_item(self, finding: dict, test):
         finding_id = finding.get("Id", "")
         title = finding.get("Title", "")
         severity = finding.get("Severity", {}).get("Label", "INFORMATIONAL").title()
+        for resource_id_arn in finding.get("Resources", []):  # adding a generic resource_id implementation separate from the below AwsEcrContainerImage one
+            description_resource_id_arn_value = resource_id_arn.get("Id")
```

Comment:
This will loop through each resource and update `description_resource_id_arn_value` to that single resource's ID, meaning only the last one ultimately gets captured. I'm assuming this isn't what you wanted to do. Maybe something like this instead to capture them all:

```suggestion
        resource_arns = []
        for resource in finding.get("Resources", []):
            if arn := resource.get("Id"):
               resource_arns.append(arn)
```